spec/interpreter/native_lib/io_spec.cr

```diff
@@ -55,4 +55,31 @@ describe "NativeLib - IO Methods" do
       itr.output.to_s.should eq("")
     end
   end
+
+  describe "#gets" do
+    it "no input" do
+      itr = interpret_with_mocked_input %q(
+        IO.gets
+      ), nil
+
+      itr.stack.pop.to_s.should eq("nil")
```

Comment:
This should probably be replaced with `itr.stack.pop.should eq(val(nil))`. The `to_s` for `TNil#to_s` might change to return a blank string, so I would like to keep things from depending on it. (in fact, the `to_s` on all of the Value classes might disappear entirely before too long. I added them long ago before native methods were really a thing.).